pygmt/tests/test_blockmean.py

```diff
@@ -26,16 +26,19 @@ def test_blockmean_input_dataframe():
     return output
 
 
-def test_blockmean_wrong_kind_of_input_table_matrix():
+def test_blockmean_input_table_matrix():
     """
     Run blockmean using table input that is not a pandas.DataFrame but still a
     matrix.
     """
     dataframe = load_sample_bathymetry()
-    invalid_table = dataframe.values
-    assert data_kind(invalid_table) == "matrix"
-    with pytest.raises(GMTInvalidInput):
-        blockmean(table=invalid_table, spacing="5m", region=[245, 255, 20, 30])
+    table = dataframe.values
+    output = blockmean(table=table, spacing="5m", region=[245, 255, 20, 30])
+    assert isinstance(output, pd.DataFrame)
+    assert output.shape == (5849, 3)
+    npt.assert_allclose(output.iloc[0], [245.888877, 29.978707, -384.0])
+
+    return output
```

Comment:
No need to return the output in tests, right?
```suggestion
    npt.assert_allclose(output.iloc[0], [245.888877, 29.978707, -384.0])
```

Reply:
All removed in https://github.com/GenericMappingTools/pygmt/pull/1280/commits/fb0d9a0d9bdc3e1e4e4a13f36cffa74a84eaf58b